sycl/plugins/unified_runtime/ur/adapters/cuda/platform.cpp

```diff
@@ -0,0 +1,203 @@
+//===--------- platform.cpp - CUDA Adapter ---------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===-----------------------------------------------------------------===//
+
+#include "platform.hpp"
+#include "common.hpp"
+#include "context.hpp"
+#include "device.hpp"
+
+#include <cassert>
+#include <cuda.h>
+#include <sstream>
+
+void enableCUDATracing();
+void disableCUDATracing();
+
+UR_DLLEXPORT ur_result_t UR_APICALL urPlatformGetInfo(
```

Comment:
why UR_DLLEXPORT here and in previous files UR_APIEXPORT ? they are the same macros, so just a nit.